src/picongpu/submit/taurus-tud/Slurm_Tutorial.rst

```diff
@@ -0,0 +1,39 @@
+Slurm examples
+==============
+
+Job Submission
+""""""""""""""
+
+PIConGPU job submission on the *Taurus* cluster at *TU Dresden*:
+
+* `tbg -s sbatch -c submit/0008gpus.cfg -t submit/taurus-tud/k80_profile.tpl $SCRATCH/runs/test123`
```

Comment:
Imo, since it's a specific example to see how to work on a HPC system on the example of Taurus, using the fast and large storage such as `$SCRATCH` is not bad.